ospray_modules/optix/Context.h

```diff
@@ -65,6 +65,8 @@ class Context
     Context();
 
     void _initialize();
+    void _printSystemInformation() const;
```

Comment:
Isn't there any way to control the amount of log output from Brayns. I tend to dislike unwanted logging information, even more when it's for developers.

Reply:
You are right, this should be using the BRAYNS_INFO output.